internal/provider/provider.go

```diff
@@ -0,0 +1,129 @@
+package ably_control
+
+import (
+	"context"
+	"os"
+
+	ably_control_go "github.com/ably/ably-control-go"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func New() tfsdk.Provider {
+	return &provider{}
+}
+
+type provider struct {
+	configured bool
+	client     *ably_control_go.Client
+}
+
+// GetSchema
+func (p *provider) GetSchema(_ context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return tfsdk.Schema{
+		Attributes: map[string]tfsdk.Attribute{
+			"token": {
+				Type:     types.StringType,
```

Comment:
Very minor, but indentation looks off to me, between this line and following lines 😅
Please ignore this comment if it is the way it should be in Go, it just may be my lack of experience with the language.

Reply:
Fixed